### PR TITLE
Change methodoly and career sections

### PR DESCRIPTION
**Pull request by Grappler2010**
1. Change list-items font-size
2. Replace the third child of the services-section
3. Replace the items of the second services-section
4. Add linkedin link for SEO
5. Change methodoly-section
6. Change career-section
7. Change email in the footer and the career-section

### CODE REVIEW

**Comment by dealwith** [comment]
image

Text: Our Vision, What sets us apart should be bigger, look at the design, it's `32px`

**Comment by dealwith** [comment]
After fixing this comment, please merge the PR

**Comment by dealwith** [comment]
Thanks for the job!

File: app/components/TeamSection/index.tsx [hunk 1]
```diff
@@ -22,7 +23,12 @@ const TeamSection = () => {
             <Image className={styles.image} src={teamSEOSrc} alt="TeamSEO" fill />
           </div>
           <div className={styles.teamTitle}>
-            <H3>{t("name")}</H3>
+            <div className={styles.socialWrapper}>
+              <H3>{t("name")}</H3>
+              <a className={styles.socialLink} href="https://www.linkedin.com/in/julian-verocai-30b1a582/">
```
Comment (dealwith) [inline app/components/TeamSection/index.tsx]: add property `target="_blank"`

**Review by dealwith — APPROVED** [review]

File: app/components/TeamSection/index.tsx (hunk 1 above)
Comment (Grappler2010) [reply app/components/TeamSection/index.tsx]: Done

**Pull request merged**